docs/how-to/build-policies/rebac/building-rebac-policies.mdx

```diff
@@ -195,6 +195,23 @@ And we can also pass, as in RBAC, an object with additonal data such as the tena
 await permit.check(userId, "assign", {type:"member_group", key:"group3", tenant:"Tenant1"} );
 ```
 
+## Visualizing your ReBAC policy - Resource Relationship Graph
+
+The resource relationship graph visually represents the connections and dependencies between various resources in your system.
+Each node in the graph represents a resource, while edges depict the relationships between them, such as ownership or permissions.
+Initially, the graph is read-only, allowing users to explore the structure and understand how resources are linked without the risk of making changes.
```

Comment:
Instead:
Initially, the graph is read-only, allowing users to explore the structure and understand how resources are linked without the risk of making changes.
I would write:
This version of the graph is read-only